Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py

```diff
@@ -207,21 +206,18 @@
         'Geometry/EcalCommonData/data/eecon.xml',
         'Geometry/EcalCommonData/data/eefixed.xml',
         'Geometry/EcalCommonData/data/eehier.xml',
-        'Geometry/EcalCommonData/data/ectkcable.xml',
         'Geometry/EcalCommonData/data/eealgo.xml',
         'Geometry/EcalCommonData/data/escon.xml',
         'Geometry/EcalCommonData/data/esalgo.xml',
         'Geometry/EcalCommonData/data/eeF.xml',
         'Geometry/EcalCommonData/data/eeB.xml',
         'Geometry/HcalCommonData/data/hcalrotations.xml',
-        'Geometry/HcalCommonData/data/PhaseI/hcalalgo.xml',
+        'Geometry/HcalCommonData/data/hcalalgo.xml',
```

Comment:
Hi @ianna : yes there is some hcal code not ported in 7X which prevents it to run - I will mention this today at the sw upgrade meeting